hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -32,6 +32,13 @@ num_grps = len(grps)
   bus_interfaces: [
     { protocol: "tlul", direction: "device" }
   ],
+  alert_list: [
+    { name: "fatal_fault",
+      desc: '''
+      This fatal alert is triggered when a fatal TL-UL bus integrity fault is detected inside the CLKMGR unit.
```

Comment:
```suggestion
      This fatal alert is triggered when a fatal TL-UL bus integrity fault is detected.
```

Also in the other units.

Reply:
yes this makes it a bit more generic - will fix (also in the other modules).